src/routes/search.py

```diff
@@ -25,17 +25,25 @@ def get_blueprint():
     return search_blueprint
 
 
-@search_blueprint.route("/search_ingredients")
+
+@search_blueprint.route("/search_ingredients", methods=["POST"])
 def search_ingredients():
-    return render_template("search.html")
+    if flask.request.method == "POST":
+        ingredients = flask.request.form.get("searchIngredients")
```

Comment:
I also recommend "import flask" or "from flask import request" at the beginning of the file to reduce calling the full library each time